src/api/authApi.ts

```diff
@@ -0,0 +1,15 @@
+import { ROUTE_CONSTANTS } from 'constants/routeConstants';
+import { URL_CONSTANTS } from 'constants/urlConstants';
+import { api, METHODS, Result } from 'utils';
+
+export const signup = (payload: Record<string, FormDataEntryValue>): Promise<Result> => (
+  api(`${URL_CONSTANTS.AUTH}${ROUTE_CONSTANTS.SIGNUP}`, METHODS.POST, JSON.stringify(payload))
```

Comment:
Подход выше позволит избежать:
1. Склеивания разных по сути констант в эндпоинт
2. Явного указания метода в параметрах

Сравни:
```suggestion
  api.post(RequestRoot.signUp, JSON.stringify(payload))
```

Под RequestRoot имею enum роутов для авторизации, написанный тут же